projects/swimlane/ngx-ui/src/lib/components/calendar/calendar.component.scss

```diff
@@ -33,7 +33,7 @@ $calendar-month-current: colors.$color-blue-grey-750;
   }
 
   button {
-    outline: 0px auto colors.$color-blue-200;
+    outline: 0px none colors.$color-blue-200;
```

Comment:
outline defaults to `none` and I'm fairly sure the same style can be written as follows:

outline: 0px colors.$color-blue-200;

That is at least what it says on the mdn documentation.

https://developer.mozilla.org/en-US/docs/Web/CSS/outline

Reply:
Does removing the style here reduce accessibility for the component in any way? Screen readers typically rely on outline and if the element can be navigated to by keyboard please ensure there is not a regression because of this change.